.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Set the default behavior
+* text=auto
+
+# sh scripts should use LF (else CRLF can cause error when running in a container on windows)
+*.sh text eol=lf
```

Comment:
Please add a new line at the end of the file.

Reply:
01ff276